decidim-core/app/views/decidim/devise/confirmations/new.html.erb

```diff
@@ -9,7 +9,7 @@
   <%= decidim_form_for(resource, namespace: "confirmation", as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
 
     <div class="form__wrapper">
-      <%= f.email_field :email, autofocus: true, value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email), autocomplete: "email", placeholder: "hi@email.com" %>
+      <%= f.email_field :email, autofocus: true, value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email), autocomplete: "email", placeholder: "hi@example.com" %>
```

Comment:
Just a question ... maybe this placeholder could be extracted to a i18n string, that could be properly localized?

Reply:
Done at 6bc3826a611d4c2686691c2f6a669df25f965e45